lib/MusicBrainz/Server/Entity/Tag.pm

```diff
@@ -13,6 +14,11 @@ sub TO_JSON {
     return shift->name;
 }
 
+sub is_genre_tag {
+    my $self = shift;
+    return ($self->name ~~ $ENTITIES{tag}{genres})
```

Comment:
Since the list is pretty big, I think we should create a hash outside this sub:
```perl
our %GENRES = map { $_ => 1 } @{ $ENTITIES{tag}{genres} };

sub is_genre_tag {
    my $self = shift;
    exists $GENRES{$self->name};
}
```